<!doctype html><html lang=en class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="This is a test repo to test out different layout and information options."><meta name=author content="Natasha Pacheco, Ph.D."><link rel=prev href=..><link href=../imaging-ai/ rel=next><link rel=icon href=../assets/images/favicon.png><meta name=generator content="mkdocs-1.5.3, mkdocs-material-9.4.4"><title>Next-Generation Sequencing ðŸ§¬ - ABCS GitHub Repository -- TEST</title><link rel=stylesheet href=../assets/stylesheets/main.bd3936ea.min.css><link rel=stylesheet href=../assets/stylesheets/palette.356b1318.min.css><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback"><style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style><script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script></head> <body dir=ltr data-md-color-scheme=default data-md-color-primary=indigo data-md-color-accent=indigo> <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> </div> <div data-md-component=announce> </div> <header class=md-header data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=.. title="ABCS GitHub Repository -- TEST" class="md-header__button md-logo" aria-label="ABCS GitHub Repository -- TEST" data-md-component=logo> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> ABCS GitHub Repository -- TEST </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> Next-Generation Sequencing ðŸ§¬ </span> </div> </div> </div> <form class=md-header__option data-md-component=palette> <input class=md-option data-md-color-media data-md-color-scheme=default data-md-color-primary=indigo data-md-color-accent=indigo aria-label="Switch to dark mode" type=radio name=__palette id=__palette_1> <label class="md-header__button md-icon" title="Switch to dark mode" for=__palette_2 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg> </label> <input class=md-option data-md-color-media data-md-color-scheme=slate data-md-color-primary=red data-md-color-accent=red aria-label="Switch to light mode" type=radio name=__palette id=__palette_2> <label class="md-header__button md-icon" title="Switch to light mode" for=__palette_1 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg> </label> </form> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=Search placeholder=Search autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg> </label> <nav class=md-search__options aria-label=Search> <button type=reset class="md-search__icon md-icon" title=Clear aria-label=Clear tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg> </button> </nav> </form> <div class=md-search__output> <div class=md-search__scrollwrap data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list role=presentation></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/abcsFrederick/abcsFrederick.TEST.github.io title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><!-- Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg> </div> <div class=md-source__repository> abcsFrederick/abcsFrederick.TEST.github.io </div> </a> </div> </nav> </header> <div class=md-container data-md-component=container> <nav class=md-tabs aria-label=Tabs data-md-component=tabs> <div class=md-grid> <ul class=md-tabs__list> <li class=md-tabs__item> <a href=.. class=md-tabs__link> About Us </a> </li> <li class="md-tabs__item md-tabs__item--active"> <a href=./ class=md-tabs__link> Next-Generation Sequencing ðŸ§¬ </a> </li> <li class=md-tabs__item> <a href=../imaging-ai/ class=md-tabs__link> Imaging AI ðŸ’» </a> </li> <li class=md-tabs__item> <a href=../statistics/ class=md-tabs__link> Statistics ðŸ“Š </a> </li> <li class=md-tabs__item> <a href=../training/ class=md-tabs__link> Training ðŸŽ“ </a> </li> <li class=md-tabs__item> <a href=../other/ class=md-tabs__link> Other ðŸ“š </a> </li> </ul> </div> </nav> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=sidebar data-md-type=navigation hidden> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary md-nav--lifted" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=.. title="ABCS GitHub Repository -- TEST" class="md-nav__button md-logo" aria-label="ABCS GitHub Repository -- TEST" data-md-component=logo> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg> </a> ABCS GitHub Repository -- TEST </label> <div class=md-nav__source> <a href=https://github.com/abcsFrederick/abcsFrederick.TEST.github.io title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><!-- Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg> </div> <div class=md-source__repository> abcsFrederick/abcsFrederick.TEST.github.io </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=.. class=md-nav__link> <span class=md-ellipsis> About Us </span> </a> </li> <li class="md-nav__item md-nav__item--active"> <input class="md-nav__toggle md-toggle" type=checkbox id=__toc> <a href=./ class="md-nav__link md-nav__link--active"> <span class=md-ellipsis> Next-Generation Sequencing ðŸ§¬ </span> </a> </li> <li class=md-nav__item> <a href=../imaging-ai/ class=md-nav__link> <span class=md-ellipsis> Imaging AI ðŸ’» </span> </a> </li> <li class=md-nav__item> <a href=../statistics/ class=md-nav__link> <span class=md-ellipsis> Statistics ðŸ“Š </span> </a> </li> <li class=md-nav__item> <a href=../training/ class=md-nav__link> <span class=md-ellipsis> Training ðŸŽ“ </span> </a> </li> <li class=md-nav__item> <a href=../other/ class=md-nav__link> <span class=md-ellipsis> Other ðŸ“š </span> </a> </li> </ul> </nav> </div> </div> </div> <div class="md-sidebar md-sidebar--secondary" data-md-component=sidebar data-md-type=toc hidden> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> </nav> </div> </div> </div> <div class=md-content data-md-component=content> <article class="md-content__inner md-typeset"> <h1>Next-Generation Sequencing ðŸ§¬</h1> <ul> <li><a href=https://github.com/CCBR/ASCENT>ASCENT</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>ASE (alternate splicing events) are identified and quantified using the ASCENT (**A**lternate **S**pli**C**ing **E**veNt **T**ools) pipeline. This workflow can be used for multi-group multi-contrasts scenarios.</li> </ul> </li> <li><a href=https://github.com/CCBR/ASAP>ASPEN</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>ASPEN or **A**tac **S**eq **P**ip**E**li**N**e is CCBR's pipeline to calls peaks for ATAC-Seq datasets. It currently accepts paired-end Illumina data and calls peak using MACS2 and Genrich peak callers.</li> </ul> </li> <li><a href=https://github.com/CCBR/CARLISLE>CARLISLE</a> <img alt=snakemake src=../images/snakemake-small.svg><ul> <li><strong>C**ut **A**nd **R**un ana**L**ys**IS</strong> pipeLin**E** calls peaks using MACS2, SEACR and GoPeaks on spike-in normalized input fastqs.</li> </ul> </li> <li><a href=https://github.com/CCBR/CCBR_circRNA_AmpliconSeq>CCBR_circRNA_AmpliconSeq</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>This is a snakemake workflow to process circRNA AmpliconSeq datasets generated to study circRNAs in KSHV and human hosts using divergent primers.</li> </ul> </li> <li><a href=https://github.com/CCBR/CHAMPAGNE>CHAMPAGNE</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li><strong>CH**rom**A**tin i**M**muno **P**recipit**A**tion sequencin**G</strong> a**N**alysis pip**E**line calls ChIPseq peaks using MACS2, GEM, and SICER followed by motif enrichment and other downstream analysis.</li> </ul> </li> <li><a href=https://github.com/CCBR/CHARLIE>CHARLIE</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>**C**ircrnas in **H**ost **A**nd vi**R**uses ana**L**ysis p**I**p**E**line detects, annotates and quantifies circRNAs in host + virus references using CIRCExplorer2, CIRI2 and many other tools in parallel.</li> </ul> </li> <li><a href=https://github.com/CCBR/CRUISE>CRUISE</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li>**C**rispr sc**R**een seq**U**encIng analy**S**is pip**E**line can be used for whole genome CRISPR KO analysis with MaGeck, Bagel and drugZ.</li> </ul> </li> <li><a href=https://github.com/CCBR/DNAnexus>DNAnexus</a><ul> <li>This repository provides access to a set of pipelines developed by CCBR members to analyze NGS data on DNAnexus. They allow a user to process raw data starting from FastQ files to reach common endpoints for downstream analysis such as a list of annotated mutations, a set of annotated peaks, or a raw counts matrix for differential expression analysis.</li> </ul> </li> <li><a href=https://github.com/NCI-RBL/Dockers/tree/main/workflows/DTEG>DTEG</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li>Integration of RNA-seq and Ribo-seq to calculate translation efficiency as the number of ribosomes per gene, normalized to transcript abundance.</li> </ul> </li> <li><a href=https://github.com/CCBR/HapFerret>HapFerret</a><ul> <li>HapFerret is an Expectation-Maximization (EM) implementation that is characterized by flexibility and ease of use, notable its use of a natural format for input genotypes and output haplotypes.</li> </ul> </li> <li><a href=https://github.com/CCBR/HERVx>HERVx</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>Human Endogenous Retrovirus expression pipeline, as known as HERVx, is a containerized pipeline to characterize retrotranscriptome. HERVx calculates Human Endogenous Retrovirus (HERV) expression in paired-end RNA-sequencing data.</li> </ul> </li> <li><a href=https://github.com/NCI-RBL/Dockers/tree/main/workflows/isomiR>ISOMIR</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li>Detection and annotation of microRNA isoforms (isomiRs).</li> </ul> </li> <li><a href=https://github.com/CCBR/l2p>l2p</a><ul> <li>List-to-pathway, or l2p, is an R package for gene set enrichment analysis that is optimized for speed.</li> </ul> </li> <li><a href=https://github.com/CCBR/METRO>METRO</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>**M**ouse n**E**oan**T**igen p**R**edict**O**r is a pipeline to characterize the effect of a mutation on an amino acid sequences and to predict the binding of peptides to any MHC molecule using netMHCpan.</li> </ul> </li> <li><a href=https://github.com/CCBR/MicroArrayPipeline>MicroArrayPipeline</a><ul> <li>MicroArray Analysis Pipeline, also known as MAAPster, is a comprehensive Shiny application and R package that performs transcriptome analysis of human or mouse Affymetrix gene expression data.</li> </ul> </li> <li><a href=https://github.com/NCI-RBL/Dockers/tree/main/workflows/MOP2>MOP2</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li>Estimation of the length of mRNA poly-A tail from Nanopore reads using Tailfindr and Nanopolish <a href=https://pubmed.ncbi.nlm.nih.gov/36723817/ >PubMed article link</a></li> </ul> </li> <li><a href=https://github.com/CCBR/RENEE>RENEE</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li><strong>R**na s**E**quencing a**N**alysis pip**E**lin**E</strong> or RENEE is an open-source, reproducible, and scalable solution for analyzing RNA-seq data. The pipeline consists of a series of data processing and quality-control steps orchestrated by Snakemake, a flexible and scalable workflow management system, to submit jobs to a cluster or cloud provider.</li> </ul> </li> <li><a href=https://github.com/NCI-RBL/Dockers/tree/main/workflows/RiboFootPrint>RiboFootPrint</a> <ul> <li>Detection of Ribosomal footprints from NGS data.</li> </ul> </li> <li><a href=https://github.com/CCBR/SINCLAIR>SINCLAIR</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li><strong>SIN**gle **C**el**L</strong> **A**nalys**I**s **R**esource or SINCLAIR is developed by the CCR Collaborative Bioinformatics Resource as an open-source, reproducible solution for multiple single cell next-generation modalities.</li> </ul> </li> <li><a href=https://github.com/CCBR/TRANQUIL>TRANQUIL</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>TRANQUIL or "TRna AbundaNce QUantification pIpeLine" is a Snakemake pipeline which quantifies tRNA using the mim-tRNAseq tool.</li> </ul> </li> <li><a href=https://github.com/CCBR/LOGAN>LOGAN</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=nextflow src=../images/nextflow-small.svg><ul> <li>LOGAN is a comprehensive whole genome-sequencing pipeline following the Broad's set of best practices.</li> </ul> </li> <li><a href=https://github.com/CCBR/XAVIER>XAVIER</a>&nbsp;&nbsp;&nbsp;&nbsp;<img alt=snakemake src=../images/snakemake-small.svg><ul> <li>e**X**ome <strong>A**nalysis and **V**ar**I**ant explor**ER</strong> aims to accurately call germline and somatic variants, to infer CNVs, and annotate variants.</li> </ul> </li> </ul> <hr> <div class=md-source-file> <small> Last update: 2023-10-18 </small> </div> </article> </div> </div> <button type=button class="md-top md-icon" data-md-component=top hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg> Back to top </button> </main> <footer class=md-footer> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <div class=md-copyright> <div class=md-copyright__highlight> Copyright &copy; 2023 ABCS </div> Made with <a href=https://squidfunk.github.io/mkdocs-material/ target=_blank rel=noopener> Material for MkDocs </a> </div> </div> </div> </footer> </div> <div class=md-dialog data-md-component=dialog> <div class="md-dialog__inner md-typeset"></div> </div> <script id=__config type=application/json>{"base": "..", "features": ["navigation.tabs", "navigation.top"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script> <script src=../assets/javascripts/bundle.94c44541.min.js></script> <script src=../js/open_in_new_tab.js></script> </body> </html>